### PR TITLE
chore: Bump up Trivy to 0.4.1

### PR DESCRIPTION
**Pull request by danielpacak**

### CODE REVIEW

File: helm/harbor-scanner-trivy/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: harbor-scanner-trivy
 version: 0.1.1
```
Comment (knqyf263) [inline helm/harbor-scanner-trivy/Chart.yaml]: Don't you have to update the chart version?

Comment (danielpacak) [reply helm/harbor-scanner-trivy/Chart.yaml]: I think it's fine just to bump up appVersion. We do not change K8S deployment descriptors.

Comment (knqyf263) [reply helm/harbor-scanner-trivy/Chart.yaml]: I think `helm upgrade` is ignored unless the version is not bumped up, but it may be my misunderstanding.

Comment (danielpacak) [reply helm/harbor-scanner-trivy/Chart.yaml]: Let me test / double check `helm upgrade` to confirm ...

Comment (knqyf263) [reply helm/harbor-scanner-trivy/Chart.yaml]: It looks fine if it works without bumping up the version.

**Pull request merged**